ligand/templates/bias_g_browser.html

```diff
@@ -0,0 +1,1710 @@
+{% extends "home/base.html" %}
+{% load static %}
+{% csrf_token %}
+
+{% block addon_css %}
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/dt/jszip-2.5.0/dt-1.10.16/b-1.5.1/b-html5-1.5.1/datatables.min.css" />
```

Comment:
Same comment as for bias_browser.html
Probably better to stick to what we have in static.
Could be good point to bring up in dev-meet. SAAS, how, where, why.

Reply:
the same as previous, of course if we had them in static, would be much better